src/solver.jl

```diff
@@ -40,19 +56,25 @@ end
 Create a solver, where the summation-by-parts (SBP) operators are of order `accuracy_order` and
 associated to the `mesh`.
 """
-function Solver(mesh, accuracy_order)
+function Solver(mesh, accuracy_order; third = false)
     if isodd(accuracy_order)
         @warn "DispersiveShallowWater.jl is expecting a central difference operator. This is not given for an odd accuracy (got $accuracy_order) order.\n This can lead to a significant reduction in the order of convergence of the solution."
     end
     D1 = periodic_derivative_operator(1, accuracy_order, mesh.xmin, mesh.xmax, mesh.N)
     D2 = periodic_derivative_operator(2, accuracy_order, mesh.xmin, mesh.xmax, mesh.N)
+    if third
+        D3 = periodic_derivative_operator(3, accuracy_order, mesh.xmin, mesh.xmax, mesh.N)
+        @assert real(D1) == real(D3)
+    else
+        D3 = nothing
+    end
```

Comment:
```suggestion
    D3 = periodic_derivative_operator(3, accuracy_order, mesh.xmin, mesh.xmax, mesh.N)
```
Creating the third order operator should not be so expensive. So I think it would be fine to create it even if we don't need it. This way the user doesn't need to specify whether they need a third order derivative operator or not. See above.